beesdoo_shift/models/res_partner.py

```diff
@@ -13,7 +13,7 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     worker_store = fields.Boolean(default=False)
-    is_worker = fields.Boolean(related="worker_store", string="Worker", readonly=False)
+    is_worker = fields.Boolean(computed="_compute_is_worker", string="Worker", readonly=True, store=True)
```

Comment:
I think that we can remove readonly attribute as it True by default for computed fields

Reply:
you don't want to remove readonly=True ?

Reply:
it has been removed with the force-push